index.html

```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
     </ul>
     </div>
 </section>
-  
+<section class="work-section">
+    <div class="tonic-container">
+        <div><img src="images/tonic-portfolio.png" alt="tonic"></div>
+        <div class="tonic-heading">Tonic</div>
+        <div class="canopy-container">
+            <div class="canopy-header">CANOPY</div>
+            <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+            <div class="canopy-text">Back End Dev</div>
+            <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+            <div class="canopy-text">2015</div>
+        </div>
+        <div class="tonic-parag">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        <div class="tonic-icons-container">
+            <ul>
+                <li><img src="images/html-tag.png" alt="html-tag"></li>
+                <li><img src="images/css-tag.png" alt="css-tag"></li> 
+                <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+           </ul>
+        </div>
+        <div class="tonic-btn">
+            <button type="button"> See Project</button>
+        </div>
+    </div>
+
+    <div class="tonic-container">
+        <div><img src="images/multi-post-portfolio.png" alt="multi-post-portfolio"></div>
+        <div class="tonic-heading">Multi-Post Stories</div>
+        <div class="canopy-container">
+            <div class="canopy-header">CANOPY</div>
+            <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+            <div class="canopy-text">Back End Dev</div>
+            <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+            <div class="canopy-text">2015</div>
+        </div>
+        <div class="tonic-parag">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        <div class="tonic-icons-container">
+            <ul>
+              <li><img src="images/html-tag.png" alt="html-tag"></li>
+              <li><img src="images/css-tag.png" alt="css-tag"></li> 
+              <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+         </ul>
+        </div>
+        <div class="tonic-btn">
+            <button type="button"> See Project</button>
+        </div>
+    </div>
+        <div class="tonic-container">
+            <div><img src="images/tonic2.png" alt="tonic"></div>
+            <div class="tonic-heading">Tonic</div>
+            <div class="canopy-container">
```

Comment:
- [ ] The project title ("Tonic") should use a header tag `hx` (Ex: `<h2>`, `<h3>` etc.) :+1: